### PR TITLE
Make vep wrapper more robust

### PR DESCRIPTION
**Pull request by apregier**
A couple tweaks that fix problems I encountered while trying to run a Trio process.

### CODE REVIEW

File: lib/perl/Genome/Db/Ensembl/Command/Run/Base.pm [hunk 1]
```diff
@@ -382,7 +382,6 @@ sub api {
     my $self = shift;
     return Genome::Db::Ensembl::Api->get_or_create(
         version => $self->ensembl_version,
-        test_name => $ENV{GENOME_SOFTWARE_RESULT_TEST_NAME},
```
Comment (tmooney) [inline lib/perl/Genome/Db/Ensembl/Command/Run/Base.pm]: Would the `GENOME_ALIGNER_INDEX_TEST_NAME` variable be useful here?  That's stretching the name of it a bit, but that's the variable for foundational results that depend on only things like the reference.

File: lib/perl/Genome/Db/Ensembl/Command/Run/Base.pm [hunk 2]
```diff
@@ -756,8 +754,7 @@ sub cache {
             $cache_result_params{sift} = 0;
         }
         $cache_result_params{users} = $self->_result_users;
-        eval {$cache_result = Genome::Db::Ensembl::VepCache->get_or_create(%cache_result_params);
-        };
+        $cache_result = Genome::Db::Ensembl::VepCache->get_or_create(%cache_result_params);
```
Comment (tmooney) [inline lib/perl/Genome/Db/Ensembl/Command/Run/Base.pm]: I'm quite glad this `eval` is gone! :smile:

File: lib/perl/Genome/Db/Ensembl/Command/Run/Base.pm (hunk 1 above)
Comment (apregier) [inline lib/perl/Genome/Db/Ensembl/Command/Run/Base.pm]: That sounds good to me!   Amendment pushed.

**Comment by tmooney** [comment]
+1 :musical_note:

**Comment by ebelter** [comment]
+1 :ghost:

**Pull request merged**